src/01-Learning-JS/00-gh-nb.md

```diff
@@ -95,9 +103,11 @@ Here's that process again for you:
 
 1. Create a multiline JS codeblock that renders to the page with the "javascript" keyword after the first 3 backticks.
 2. Inside the "javascript" codeblock, add the following text `// I will show up on the page`.
+```java script `//I will show up on the page`
     - **NOTE**: This codeblock should render on the locally hosted page.
 3. Create another multiline JS codeblock that is executable by using the "js" keyword after the first 3 backticks.
 4. Inside the "js" codeblock, add the following text `console.log("I do NOT show up on the page")`.
+```js `console.log("i do NOT show up on the page")`
```

Comment:
Should be the following, but without the commented out parts. I needed to do that so the suggestion should you the code.

```suggestion
// ```js
// console.log("i do NOT show up on the page")
// ```
```